Lib/ufo2ft/postProcessor.py

```diff
@@ -165,11 +165,8 @@ def process_glyph_names(self, useProductionNames=None):
             else:
                 self.set_post_table_format(self.otf, 3.0)
 
-    def _rename_glyphs_from_ufo(self):
```

Comment:
keep this instance method `_rename_glyphs_from_ufo(self)` where it was and define the new `rename_glyphs` staticmethod and use the latter in the former